src/Dapper.FluentMap.Dommel/Mapping/DommelEntityMap.cs

```diff
@@ -44,5 +44,15 @@ protected void ToTable(string tableName)
         {
             TableName = tableName;
         }
+
+        /// <summary>
+        /// Sets the table name for the current entity.
+        /// </summary>
+        /// <param name="tableName">The name of the table in the database.</param>
+        /// <param name="schemaName">The name of the table schema in the database.</param>
+        protected void ToTable(string tableName, string schemaName)
```

Comment:
I would have said yes if there was not a second method with the same name and only tableName parameter.
If we look in EfCore : https://github.com/aspnet/EntityFramework/blob/master/src/Microsoft.EntityFrameworkCore.Relational/RelationalEntityTypeBuilderExtensions.cs
They choose to put schema as second parameter too.